packages/components/src/components/linear-progress/linear-progress.tsx

```diff
@@ -1,7 +1,13 @@
 import classNames from 'classnames';
 import React from 'react';
 
-export const LinearProgress = ({ progress }) => (
+type TLinearProgress = {
+    progress: number;
+    className: string;
+    height: number;
+};
+
+export const LinearProgress = ({ progress }: Partial<TLinearProgress>) => (
     <div className={classNames('dc-linear-progress')}>
         <div className={classNames('dc-linear-progress__bar')} style={{ width: `${progress}%` }} />
```

Comment:
Is this classNames required?
  <div className='dc-linear-progress__bar' style={{ width: `${progress}%` }} />